### PR TITLE
update 2.3 Changelog

### PR DESCRIPTION
**Pull request by Swiftb0y**
I went through all PR to 2.3 since May 8. (which was when @Holzhaus updated the changelog the last time). I might have missed some stuff or be overly eager to include others (I tried to include every PR since the last changelog update).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -58,6 +62,10 @@
 * Fix caching of duplicate tracks that reference the same file [#3027](https://github.com/mixxxdj/mixxx/pull/3027)
 * Use 6 instead of only 4 compatible musical keys (major/minor) [#3205](https://github.com/mixxxdj/mixxx/pull/3205)
 * Fix possible crash when trying to refocus the tracks table while another Mixxx window has focus [#3201](https://github.com/mixxxdj/mixxx/pull/3201)
+* Don't create new tags in file when exporting metadata to it [#3898](https://github.com/mixxxdj/mixxx/pull/3898)
+* Fix playlist files beginning with non-english characters not being loaded: [#3916](https://github.com/mixxxdj/mixxx/pull/3916)
```
Comment (Holzhaus) [inline CHANGELOG.md:66]:
```suggestion
* Fix playlist files beginning with non-english characters not being loaded [#3916](https://github.com/mixxxdj/mixxx/pull/3916)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -58,6 +62,10 @@
...
+* Enable sorting in "Hidden Tracks" and "Missing Tracks" views [#3828](https://github.com/mixxxdj/mixxx/pull/3828) [lp:1828555](https://bugs.launchpad.net/mixxx/+bug/1828555/) [lp:1924616](https://bugs.launchpad.net/mixxx/+bug/1924616/)
+* fix Track table being empty after start: "[#3935](https://github.com/mixxxdj/mixxx/pull/3935/) [lp:1930546](https://bugs.launchpad.net/mixxx/+bug/1930546/) [lp:1924843](https://bugs.launchpad.net/mixxx/+bug/1924843/)"
```
Comment (Holzhaus) [inline CHANGELOG.md:68]:
```suggestion
* Fix track table being empty after start [#3935](https://github.com/mixxxdj/mixxx/pull/3935/) [lp:1930546](https://bugs.launchpad.net/mixxx/+bug/1930546/) [lp:1924843](https://bugs.launchpad.net/mixxx/+bug/1924843/)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -105,6 +114,9 @@
 * Don't trash user configuration if an error occurs when writing [#3192](https://github.com/mixxxdj/mixxx/pull/3192)
 * Enable CUE sheet recording by default [#3374](https://github.com/mixxxdj/mixxx/pull/3374)
 * And countless other small fixes and improvements (too many to list them all!)
+* Fix crash when right double clicking GLSL waveforms [#3904](https://github.com/mixxxdj/mixxx/pull/3904)
```
Comment (Holzhaus) [inline CHANGELOG.md:117]:
```suggestion
* Fix crash when double clicking GLSL waveforms with right mouse button [#3904](https://github.com/mixxxdj/mixxx/pull/3904)
```

**Comment by Swiftb0y** [comment]
> Thanks. Can you update the changelog in the manual as well when this has been merged?

Can do.

**Comment by ronso0** [comment]
track info on double click
https://github.com/Swiftb0y/mixxx/pull/4

File: CHANGELOG.md [hunk 4]
```diff
@@ -71,6 +79,7 @@
 * Fix possible memory corruption using JACK on Linux [#3160](https://github.com/mixxxdj/mixxx/pull/3160)
 * Fix changing of vinyl lead-in time [lp:1915483](https://bugs.launchpad.net/mixxx/+bug/1915483) [#3781](https://github.com/mixxxdj/mixxx/pull/3781)
 * Fix tempo change of non-const beatgrid track on audible deck when cueing another track [#3772](https://github.com/mixxxdj/mixxx/pull/3772)
+* Fix crash when changing effect unit routing [#3882](https://github.com/mixxxdj/mixxx/pull/3882) [lp:1775497](https://bugs.launchpad.net/mixxx/+bug/1775497)
```
Comment (ronso0) [inline CHANGELOG.md:82]:
```suggestion
* Fix crash when changing effect unit routing [#3882](https://github.com/mixxxdj/mixxx/pull/3882) [lp:1775497](https://bugs.launchpad.net/mixxx/+bug/1775497)
* Make microphone ducking use strength knob the same way in automatic & manual mode [#2750](https://github.com/mixxxdj/mixxx/pull/2750)
```

**Comment by daschuer** [comment]
The addition of AAC-HE streaming is also important: https://github.com/mixxxdj/mixxx/pull/3615

Did you go to the PRs by sorting them in GitHub? They are sorted by the date when they where opened, so I am afraid you miss some old ones that have been merged finally. 

Alternative you may filter the git log by "Merge pull request" which gives you a list of PRs sorted by merge date.

**Comment by Holzhaus** [comment]
For that reason, i used git log and only showed merge commits when I updated the changelog back in May.

**Comment by Swiftb0y** [comment]
> Did you go to the PRs by sorting them in GitHub? They are sorted by the date when they where opened, so I am afraid you miss some old ones that have been merged finally.
> Alternative you may filter the git log by "Merge pull request" which gives you a list of PRs sorted by merge date.

No I used `is:pr is:closed base:2.3 merged:>2021-05-08`. So I should've gotten all PRs since the last changelog update by Jan. All the PRs which I didn't include were merged before May because I figured they were already included in the last update.

**Comment by ywwg** [comment]
thanks for doing this!

**Pull request merged**